src/components/reviews-by-category.jsx

```diff
@@ -1,4 +1,40 @@
-const ReviewsByCategory = () => {
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { getReviews } from "../utils/api";
+import ReviewCard from "../utils/review-card";
 
-}
-export default ReviewsByCategory
+const ReviewsByCategory = (props) => {
+  const [currReviews, setCurrReviews] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const { category } = useParams();
+  const { currCategories } = props;
+
+  const categoryObj = currCategories.filter((elem) => {
+    return elem.slug === category;
+  })[0];
```

Comment:
Like Doug mentioned, the .find method is ideal for this situation when you know you only want one element from an array